### PR TITLE
fixed epochStart integration tests by reducing the sleep inside tests

### PR DESCRIPTION
**Pull request by raduchis**
fixed epochStart integration tests that failed with panic: test timed out after 10m0s by reducing the sleep inside the tests

fixed TestEpochValidatorInfoCreator_CreateValidatorInfoMiniBlocksShouldBeCorrect that failed sometimes becaue of a range on a map

### CODE REVIEW

File: epochStart/metachain/validators.go [hunk 1]
```diff
@@ -63,7 +63,17 @@ func (r *validatorInfoCreator) CreateValidatorInfoMiniBlocks(validatorInfo map[u
...
+	shardIds := make([]uint32, 0)
+	for k := range validatorInfo {
+		shardIds = append(shardIds, k)
+	}
+
+	sort.Slice(shardIds, func(a, b int) bool {
+		return shardIds[a] < shardIds[b]
+	})
+
+	for _, shardId := range shardIds {
```
Comment (sasurobert) [inline epochStart/metachain/validators.go]: for i:=0, i < r.shardCoordinator.NumberOfShards(); i++

File: epochStart/metachain/validators.go [hunk 2]
```diff
@@ -63,7 +63,17 @@ func (r *validatorInfoCreator) CreateValidatorInfoMiniBlocks(validatorInfo map[u
...
+		validators := validatorInfo[shardId]
```
Comment (sasurobert) [inline epochStart/metachain/validators.go]: + metachain separately.

File: epochStart/metachain/validators.go [hunk 3]
```diff
@@ -63,7 +63,17 @@ func (r *validatorInfoCreator) CreateValidatorInfoMiniBlocks(validatorInfo map[u
 
 	miniblocks := make([]*block.MiniBlock, 0)
 
-	for _, validators := range validatorInfo {
+	shardIds := make([]uint32, 0)
+	for k := range validatorInfo {
+		shardIds = append(shardIds, k)
+	}
+
+	sort.Slice(shardIds, func(a, b int) bool {
```
Comment (sasurobert) [inline epochStart/metachain/validators.go]: no need for this.

File: epochStart/metachain/validators_test.go [hunk 4]
```diff
@@ -105,6 +106,7 @@ func verifyMiniBlocks(bl *block.MiniBlock, infos []*state.ValidatorInfo, marshal
 		bl.ReceiverShardID != core.AllShardId ||
 		len(bl.TxHashes) == 0 ||
 		bl.Type != block.PeerBlock {
+		log.Warn("verifyMiniBlocks existed on first if")
```
Comment (bogdan-rosianu) [inline epochStart/metachain/validators_test.go]: change log message. "first if" doesn't tell that much.. maybe "miniblock is not of PeerBlock type"

File: epochStart/metachain/validators_test.go [hunk 5]
```diff
@@ -118,8 +120,10 @@ func verifyMiniBlocks(bl *block.MiniBlock, infos []*state.ValidatorInfo, marshal
 		vi := validatorCopy[i]
 		unmarshaledVi := &state.ValidatorInfo{}
 		_ = marshalizer.Unmarshal(unmarshaledVi, txHash)
-
 		if !reflect.DeepEqual(unmarshaledVi, vi) {
+			log.Warn("verifyMiniBlocks existed on second if",
```
Comment (bogdan-rosianu) [inline epochStart/metachain/validators_test.go]: change this log message to something that suggests wrong type

File: epochStart/metachain/validators.go (hunk 1 above)
Comment (raduchis) [reply epochStart/metachain/validators.go]: fixed

File: epochStart/metachain/validators.go (hunk 2 above)
Comment (raduchis) [reply epochStart/metachain/validators.go]: fixed

File: epochStart/metachain/validators.go (hunk 3 above)
Comment (raduchis) [reply epochStart/metachain/validators.go]: removed

File: epochStart/metachain/validators_test.go (hunk 4 above)
Comment (raduchis) [reply epochStart/metachain/validators_test.go]: removed. It was just for testing purposes

File: epochStart/metachain/validators_test.go (hunk 5 above)
Comment (raduchis) [reply epochStart/metachain/validators_test.go]: removed. was just for testing purposes

**Pull request merged**